### ISSUE TITLE
Can't get to front page

### ISSUE THREAD

**Issue by mhjb** [loomio/loomio#4211]
I've signed out (which seemed to take a couple of tries). Now when I visit loomio.org I'm redirected to the dashboard with a sign-in modal:

![image](https://user-images.githubusercontent.com/1380991/28236176-14292c44-6973-11e7-9cf8-e31ee2fe2e92.png)

**Comment by pqrkchqps** [comment]
I created a fork of your project to start my own company inspired by social values.  I noticed that your homepage is down and thought I would mention what I did to get https://diehard.fund working (currently no new features), and introduce my project to loomio.  

Simple Fix For Front Page
My development environment needed to delete the plugins folder and manually create the symlink in order to load the plugins.


Introduction (If you are interested)
Dreamer Assist Corporation (http://www.dreamer-assist.com/ WIP) is attempting to end poverty (at least make better) through social business.  Technological progress has the ability to automate a lot of jobs and make life easier, but many people are still trapped at minimum wage and can't get ahead (or unemployed).  Dreamer Assist Corporation is about using technology to free us from unproductive and unfulfilling work and facilitating others in pursuing their 'happiness'.  My first product Diehard.Fund is complicate crowdfunding platform that allows gamers to become game designers, game developers or game investors using a new business paradigm where users drive the design....  I will write up some better docs, but I thought Loomio deserved an explanation for my re-branding of the Loomio platform into a crowdfunding app.
